lenses/connect-customlens-performanceEfficiency.json

```diff
@@ -28,8 +28,8 @@
 							"id": "choice2",
 							"title": "Mechanisms exist to monitor and report on the metrics",
 							"helpfulResource": {
-							"displayText": "Amazon Connect provides suprivsors and admins the ability to monitor Contact Center metrics with out-of-the box reporting cababilites. If organizations would like to use third party reporting and analytics tools, a mechanism should be implemented to stream data from Connect to third party app(s).",
-							"url": "https://docs.aws.amazon.com/connect/latest/adminguide/amazon-connect-metrics.html"
+								"displayText": "Amazon Connect provides suprivsors and admins the ability to monitor Contact Center metrics with out-of-the box reporting cababilites. If organizations would like to use third party reporting and analytics tools, a mechanism should be implemented to stream data from Connect to third party app(s).",
```

Comment:
Not sure why these are showing up as weird tab changes. These sections were un-edited.